src/components/Form/ActionButton/Prompt.css

```diff
@@ -1,11 +1,17 @@
 .toggle-comment-button {
   position: absolute;
-  right: 1.5em;
+  right: 0;
+  font-weight: bold;
+  color: #aaa;
+  font-style: italic;
 }
 
 button.light.toggle-comment-button {
   background-color: transparent;
 }
+.panel > label {
```

Comment:
this css rule is quite generic and might conflict with over part of the app. I'd rather use a css class to ensure there is no side effect.